### DISCUSSION TITLE
Show Github stars of some other repo while directing edit/view button another repo

### DISCUSSION THREAD

**Discussion by jonaprieto** [squidfunk/mkdocs-material#5385] [Q&A] [answered]
Say you have your documentation files in a different repository X than the one you have for your main project, repository Y. How can I still get the Github stars from repository Y:

![image](https://user-images.githubusercontent.com/1428088/232467716-d8959de8-0265-47ba-ac26-5fcc99b6f20a.png)

But linking the following buttons to repository X.

![image](https://user-images.githubusercontent.com/1428088/232467788-11008dda-1016-46c0-af8e-bf102ff61ba6.png)

**Comment by squidfunk** [comment, marked as answer]
According to the [documentation of MkDocs](https://www.mkdocs.org/user-guide/configuration/#edit_uri), you can just use an absolute value for `edit_uri`, pointing it to another repo:

> `edit_uri` can actually be just an absolute URL.
